plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/MultiParentCompileHook.java

```diff
@@ -85,7 +85,7 @@ public void validate(Map<String, Object> toCheck) throws Exception {
     @Override
     public boolean check(Map<String, Object> info) throws Exception {
         return BlueOceanHook.isBOPlugin(info) || DeclarativePipelineHook.isDPPlugin(info) || StructsHook.isStructsPlugin(info) ||
-        ConfigurationAsCodeHook.isCascPlugin(info) || PipelineRestApiHook.isPipelineStageViewPlugin(info);
+        SwarmHook.isSwarmPlugin(info) || ConfigurationAsCodeHook.isCascPlugin(info) || PipelineRestApiHook.isPipelineStageViewPlugin(info);
```

Comment:
We should finally rewrite the hook engine at some point to an extension-alike approach.
This line makes me cry :D

Reply:
Agreed, this would be a good area for future improvement.